### PR TITLE
Allow ^6.0 for symfony/*

### PR DESCRIPTION
**Pull request by franmomu**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | 

#### Summary

Follow-up of https://github.com/doctrine/mongodb-odm/pull/2370 adding a job to test it: https://github.com/doctrine/mongodb-odm/pull/2375/checks?check_run_id=3824035674

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
         "jean85/pretty-package-versions": "^1.3.0 || ^2.0.1",
         "mongodb/mongodb": "^1.2.0",
         "psr/cache": "^1.0 || ^2.0 || ^3.0",
-        "symfony/console": "^3.4 || ^4.1 || ^5.0",
+        "symfony/console": "^3.4 || ^4.1 || ^5.0 || ^6.0",
```
Comment (IonBazan) [inline composer.json:38]: Maybe we could drop <4.4 with this release? 3.4 is already EOL.
Perhaps we can do that in separate PR and see if there are any BC layers we can remove.

**Review by IonBazan — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (SenseException) [reply composer.json:38]: I guess this is a good opportunity to drop 3.4 - 4.3.

**Review by IonBazan — APPROVED** [review]

**Review by malarzm — APPROVED** [review]

**Comment by malarzm** [comment]
Thanks @franmomu for picking this up 🎉
